### PR TITLE
Create STACK programs in Python

### PR DESCRIPTION
**Pull request by ashwek**

### CODE REVIEW

File: STACKS/Python/balancedParanthesis.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            stack.append(i)
+        elif( i in ']})' ):
+            if( len(stack) == 0 ):
+                return False;
+
+            if( stack[-1] == brackets[i] ): #if bracket is same as the top element of the stack
+                stack.pop()
+
+    return len(stack) == 0
+
```
Comment (GauravWalia19) [inline STACKS/Python/balancedParanthesis.py]: Add one more line..

File: STACKS/Python/balancedParanthesis.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+def balancedParanthesis(Text):
```
Comment (GauravWalia19) [inline STACKS/Python/balancedParanthesis.py]: Here Text should be text as parameters are always small

File: STACKS/Python/balancedParanthesis.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    stack = [] # Create empty list (stack)
+    brackets = {'}':'{', ']':'[', ')':'('}
+
+    for i in Text:
+        if( i in '[{(' ):
```
Comment (GauravWalia19) [inline STACKS/Python/balancedParanthesis.py]: Remove brackets as I think its not a good way to use brackets in py

**Comment by ashwek** [comment]
Made necessary changes.
i couldn't find any indentation bugs, if it still exists could you please specify where you found it. 👍

**Comment by anarchymonkey** [comment]
thank you for your contribution 👍

**Comment by anarchymonkey** [comment]
@ashwek  the indentation is perfect , cause if it was not the compiler will give errors. So i didnt see any problem in it. keep on contributing and encourage your friends too. Thank you 💯 🥇

**Pull request merged**